cloud

```diff
@@ -624,7 +630,15 @@ function cloud_functions_deploy() {
                           --entry-point kcidb_load_queue \
                           --env-vars-file "$env_yaml" \
                           --runtime python37 \
-                          --trigger-topic "${trigger_topic}" \
+                          --trigger-topic "${load_queue_trigger_topic}" \
+                          --memory 256MB \
+                          --timeout 540
+    cloud_function_deploy "$project" "${prefix}pick_notification" \
+                          "$source" \
+                          --entry-point kcidb_pick_notifications\
+                          --env-vars-file "$env_yaml" \
+                          --runtime python37 \
+                          --trigger-topic "${load_queue_trigger_topic}" \
```

Comment:
This still has to be triggered off your new trigger topic (`pick_notifications_trigger_topic`), not `load_queue_trigger_topic`, it's what we created it for, right?